languages/python/sqlalchemy-oso/tox.ini

```
@@ -3,6 +3,7 @@ skip_missing_interpreters=true
 envlist = {py3,pypy3}-sqlalchemy{13,14}-{earliest,latest}
 
 [testenv]
+passenv = *
```

assuming we're doing this for the new postgres variables, how about this?

```suggestion
passenv = POSTGRES_*
```

https://tox.readthedocs.io/en/latest/config.html?highlight=passenv#conf-passenv

would feel better keeping the test envs as clean & isolated as possible

---

oh cool